src/postgres-util/src/tunnel.rs

```diff
@@ -119,7 +119,7 @@ pub const DEFAULT_SNAPSHOT_STATEMENT_TIMEOUT: Duration = Duration::ZERO;
 ///
 /// This wraps [`tokio_postgres::Config`] to allow the configuration of a
 /// tunnel via a [`TunnelConfig`].
-#[derive(Debug, Clone)]
+#[derive(Clone)]
```

Comment:
Why did this lose its `Debug` formatting?

Reply:
it shouldnt have!